docs/coordinates/galactocentric.rst

```diff
@@ -0,0 +1,108 @@
+.. _coordinates-galactocentric:
+
+==========================================
+Transforming to Galactocentric coordinates
+==========================================
+
+[sec:transform]
```

Comment:
This appears plain text in the HTML output ... was it supposed to be some RST markup?